railties/lib/rails/commands/server/server_command.rb

```diff
@@ -275,22 +259,6 @@ def prepare_restart
           FileUtils.rm_f(options[:pid]) if options[:restart]
         end
 
-        def deprecate_positional_rack_server_and_rewrite_to_option(original_options)
-          if using
-            ActiveSupport::Deprecation.warn(<<~MSG)
-              Passing the Rack server name as a regular argument is deprecated
```

Comment:
This deprecation added by Rails 6.0. Ref: #32058

Reply:
I reverted fa791fb8e2a718b5d0430c7ca5a454678dfc192d with eb63faaa1af7ef28ae1a716d068acc447e28c174.

Reply:
Thank you for catching that.